### PR TITLE
Improved image carousel on startPage

### PR DESCRIPTION
**Pull request by OliviaIsberg**
Improved image carousel on startPage

### CODE REVIEW

**Review by sabinaander — APPROVED** [review]
Mycket snyggare ratio :)

File: src/components/Carousel.tsx [hunk 1]
```diff
@@ -43,13 +43,15 @@ function ShowCarousel() {
...
 function ShowImages(props: Props) {
   return (
-    <Paper sx={{ height: '35vw', width: '100%' }}>
-      <img
-        style={{ height: '100%', width: '100%' }}
-        src={props.item.image}
-        alt=""
-      />
-    </Paper>
+    // <Paper sx={{ width: '100%' }}>
```
Comment (spaceflake) [inline src/components/Carousel.tsx]: Ta bort?

Comment (OliviaIsberg) [reply src/components/Carousel.tsx]: yes hade missat xD

**Review by spaceflake — APPROVED** [review]
Toppen!

**Pull request merged**